composer.json

```diff
@@ -70,7 +70,8 @@
         "friendsofphp/php-cs-fixer": "~2.16.0",
         "kunalvarma05/dropbox-php-sdk": "^0.2.1",
         "tightenco/collect": "5.2.*",
-        "ampache/ampacheapi-php": "^1.0"
+        "ampache/ampacheapi-php": "^1.0",
+        "ext-json": "*"
```

Comment:
i'll actually put this in the installer tests as well

Reply:
PHPStorm had been complaining about it for a while now.